### PR TITLE
Update advantages in Migrate to new dashboard

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/dashboards/dashboards-migration/index.md [hunk 1]
```diff
@@ -9,10 +9,12 @@ The **Migrate to Dashboards** functionality allows you to transform **Dashboards
 
 ## Advantages
 
-**Dashboards**
-
-- Persistent release and update with new feature additions.
-- Provides the unique ability to analyze metrics metadata and log data on the same dashboard in an integrated seamless view. This is what you need to monitor and manage a Kubernetes environment effectively.
-- You can control the visual display of metric and log data.
-- Streamline dashboard configuration and on-the-fly analytic visualizations with its new templating features.
+- Visulize queries with new dashboard panels, including honeycombs, bubble charts, heat maps, and funnel charts.
```
Comment (oshoghi) [inline docs/dashboards/dashboards-migration/index.md:12]: Visualize has a typo here. Can you also link to  https://help.sumologic.com/docs/dashboards/panels/

Comment (JV0812) [reply docs/dashboards/dashboards-migration/index.md]: Thanks @oshoghi. I have fixed your comment.

File: docs/dashboards/dashboards-migration/index.md [hunk 2]
```diff
@@ -9,10 +9,12 @@ The **Migrate to Dashboards** functionality allows you to transform **Dashboards
...
-- You can control the visual display of metric and log data.
-- Streamline dashboard configuration and on-the-fly analytic visualizations with its new templating features.
+- Visualize queries with new [dashboard panels](/docs/dashboards/panels/), including honeycombs, bubble charts, heat maps, and funnel charts.
+- Provides the unique ability to analyze metrics metadata and log data on the same dashboard in an integrated seamless view.
+- Visualize multiple queries in a single panel.
+- Provides streamlined dashboard configuration with inline editing.
+- Includes powerful [template variables](/docs/dashboards/filter-template-variables/) that allows you to alter your queries in multiple ways.
+- Supports [scheduled reports](/docs/dashboards/scheduled-report/), that delivers the report right to your inbox.
+- Modern user interface with enhanced user experience and performance.
+- Persistent support for new features, updates, and security enhancements.
```
Comment (jpipkin1) [inline docs/dashboards/dashboards-migration/index.md:19]:
```suggestion
- Lets you visualize queries with new [dashboard panels](/docs/dashboards/panels/), including honeycombs, bubble charts, heat maps, and funnel charts.
- Provides the unique ability to analyze metrics metadata and log data on the same dashboard in an integrated seamless view.
- Allows you to visualize multiple queries in a single panel.
- Provides streamlined dashboard configuration with inline editing.
- Includes powerful [template variables](/docs/dashboards/filter-template-variables/) that allow you to alter your queries in multiple ways.
- Supports [scheduled reports](/docs/dashboards/scheduled-report/) that deliver the report right to your inbox.
- Has a modern user interface with enhanced user experience and performance.
- Includes persistent support for new features, updates, and security enhancements.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@JV0812, I just have some minor wording suggestions.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**